src/Scripts/Game/Components/Vehicle/EL_LicensePlateManagerComponent.c

```diff
@@ -1,12 +1,14 @@
 [EntityEditorProps(category: "EL/LicnesePlateManagerComponent", description: "This renders license plates on your vehicle.", color: "0 0 255 255")]
-class EL_LicensePlateManagerComponentClass: ScriptComponentClass
+class EL_LicensePlateManagerComponentClass : ScriptComponentClass
 {
-};
+}
 
-class EL_LicensePlatePointInfo: PointInfo
+class EL_LicensePlatePointInfo : PointInfo
 {	
 	EL_LicensePlateEntity m_Object;
 	
+	//------------------------------------------------------------------------------------------------
+	//! Deconstructor
 	void ~EL_LicensePlatePointInfo()
```

Comment:
Destructor should be at the bottom

> Methods should be sorted in the following order (top to bottom):
> General methods
> EOnFrame
> EOnInit
> Constructor
> Destructor

Reply:
In this case it is the only method in the class hence it is last. Do you want to move the whole class to the bottom of the file?

Reply:
Oh sorry didn't see that. No need to move it to the bottom.